detection-rules/attachment_adobe_image_lure_qr_code.yml

```diff
@@ -15,6 +15,9 @@ source: |
           or any(ml.logo_detect(beta.message_screenshot()).brands,
                  .name == "Adobe"
           )
+          or any(file.explode(.),
```

Comment:
Given it's against images or pdfs, do you need leverage the OCR output?

Reply:
We should be ok here, OCR output of the root attachment becomes a deeper `text/plain` object in the FileExplode output that has the OCR output in `.scan.strings.strings`

Reply:
So I guess, this _is_ using the OCR output, just not directly.